docs/commands/index.md

```diff
@@ -9,7 +9,6 @@ Documentation commands:
 - [`preview`](preview.md) Start a local preview of a Redocly project with one of the product NPM packages.
 - [`translate`](translate.md) Generate translation keys for a Redocly Realm, Reef, or Revel project.
 - [`eject`](eject.md) Eject and modify components from the core theme in a Redocly Realm, Reef, or Revel project.
-- [`preview-docs`](preview-docs.md) Preview API reference docs for the specified API description.
```

Comment:
I'd still like this to remain as many will use CLI v1 for some time. Maybe we need some kind of v1/v2 docs?